tox.ini

```diff
@@ -1,6 +1,6 @@
 # to run for a specific environment, use ``tox -e ENVNAME``
 [tox]
-envlist = py37,py38,py39,py310,py311,pypy3,docs
+envlist = py37,py38,py39,py310,py311,pypy3,docs,nopytz
```

Comment:
Did we add py312 in another PR? If not, then that should be a new issue. If it was, then let's merge main into this branch.

Reply:
We did not add that. py37 should be removed. - new PR...